src/app/view/view.js

```diff
@@ -0,0 +1,134 @@
+import { BUTTON_TEXT, DOM, WINNER_ALERT_TIMEOUT_AMOUNT } from '../../lib/constants.js';
+import { createDivWithClassName, createElementWithId, selectDOM } from '../../lib/utils.js';
+
+class RacingCarGameView {
+  constructor() {
+    this.#initDOM();
+  }
+
+  #initDOM() {
+    this.countInputForm = selectDOM(`#${DOM.COUNT_INPUT_FORM_ID}`);
+    this.gameStartBtn = selectDOM(`#${DOM.GAME_START_BTN_ID}`);
+    this.resultField = selectDOM(`#${DOM.RESULT_FIELD_ID}`);
+    this.gameProgress = selectDOM(`#${DOM.GAME_PROGRESS_ID}`);
+    this.winners = selectDOM(`#${DOM.WINNERS_ID}`);
+  }
+
+  #initSpinner() {
+    this.rotateSpinner();
+    this.currentDegrees = 0;
+  }
+
+  renderGameStart(cars) {
+    this.disableGameStartBtn();
+    const progressTemplate = cars.map((car) => RacingCarGameView.generateProgressTemplate(car));
+
+    this.gameProgress.append(...progressTemplate);
+    this.spinners = this.gameProgress.querySelectorAll(`.${DOM.SPINNER_CLASS}`);
+    this.stepContainers = this.gameProgress.querySelectorAll(`.${DOM.STEP_CONTAINER_CLASS}`);
+    this.#initSpinner();
+  }
+
+  renderRoundResult(movedCars, totalRounds, finishedRounds) {
```

Comment:
```suggestion
  renderRoundResult(movedCars, totalRounds, currentRound) {
```
현재 라운드가 더 맞는 네이밍인 것 같은데 어떨까요?

Reply:
이 부분을 더 자세히 들여다 보니 근본적으로 리팩터링 할 수 있는 방법이 생각나서 아예 수정했습니다.

지금 저 두 인자를 받아서 하는 것이 두 가지가 동일한 지를 비교하는 게 전부인데 
굳이 두 인자를 받아올 필요가 없다는 걸 발견했습니다.
그래서 애초에 컨트롤러 부분에서 뷰로 넘어올 때
두 변수를 비교해서 게임이 종료되었는지 여부만 넘겨주도록 수정했습니다.

Reply:
```js
// 정의 부분
renderRoundResult(movedCars, isGameOver) {
...
  if (isGameOver) {
    this.#removeSpinner();
  }
}

// 호출 부분
const isGameOver = this.totalRounds === this.finishedRounds;
this.view.renderRoundResult(movedCars, isGameOver);
```

Reply:
LGTM~